_store/umws-applewatch-app.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Apple Watch App"
+name: Apple Watch App
+intro: "Track your kimai times directly at your wrist!"
+developer: umws
+date: "2020-01-22 20:00:00 +0200"
+image: https://kimaiapp.umws.de/images/screens/kimaiTracker_tracking.png
+icon: fab fa-apple
+price: 1,92€
+version: 1.0
+download: https://apps.apple.com/app/kimaitracker/id1494495700
+new: true
+email: kimaiapp@umws.de
+tags:
+  - app
+  - Apple Watch
+---
+
+Start your Kimai time tracking right at your Apple Watch now!
+
+### Features
+
+* Start/Stop tracking
+* Set favourites for regular used Activities
+* See recent tracked Activities
+* Browse your Customers, Projects and Activities
+* Complications to show current tracking
+
+
+**Note**:
+This app is not for offline tracking. It requires a connection to the API of your kimai server (v0.9 or higher - API v0.3).
+
+<p style="text-align:center"><a href="https://kimaiapp.umws.de">kimaiapp.umws.de</a></p>
+
+### App Screens
+
+<p style="text-align:center">Use your <strong>favourite</strong> or <strong>recent</strong> activities to start tracking immediately</p>
+<p style="text-align:center">
+    <img src="https://kimaiapp.umws.de//images/screens/kimaiTracker_favourites.png" width="200" style="margin:0 10px">
```

Comment:
`kimaiapp.umws.de//images` vs `kimaiapp.umws.de/images`

Reply:
Both should work, even if it was not my intention to add the double //